go-runtime/compile/external-module-template/_ftl/go/modules/{{ range .NonMainModules }}{{ push .Name . }}{{ end }}/external_module.go

```diff
@@ -12,13 +12,13 @@ import (
 var _ = context.Background
 
 {{- range .Decls }}
-{{- if .IsExported}}
-{{if and (is "Enum" .) .IsValueEnum}}
-{{$enumName := .Name -}}
-{{if .Comments -}}
-{{.Comments|comment}}
+{{if .Comments}}
```

Comment:
I know @alecthomas just told you to do it this way, but does this introduce a bug where non-exported decls have their comments put in `external_module.go` but they are there dangling without their declaration?

Reply:
If that does happen, we should still do it this way but also check `.IsExported`

Reply:
But I would think if that were the case we would have seen them in the tests...